pkg/addons/data/DaemonSet.apps/openshift-azure-monitoring/etcd-metrics.yaml

```diff
@@ -36,7 +36,7 @@ spec:
         - -servingcert
         - /var/serving-cert/tls.crt
         - -whitelist
-        - /metrics(/|$)
+        - ^/metrics(/|$)
```

Comment:
This should not work. It should get `https://master-0000:2789/metrics` as `req.URL.String()`. Will test in an hour